taxi/tests.py

```diff
@@ -1,3 +1,89 @@
 from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from taxi.models import Manufacturer, Car
 
-# Create your tests here.
+
+class IndexViewTest(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="testuser", password="password123"
```

Comment:
Ensure that the password is hashed by using `get_user_model().objects.create_user()` instead of directly assigning a plain text password.